sounds/f_q.md

```diff
@@ -27,7 +27,7 @@ Q
 * `KWEUBG`: quick
 * `KWABG`: quack
 * `KWEUZ`: quiz
-* `KWEUP`: quip
+* `KWEUP`: equip
```

Comment:
I think I'd prefer to remove this line completely, or move it to briefs

Reply:
heh I didn't realize there was already a push on this one, agreed, move it to briefs.